app/test_main.py

```diff
@@ -1 +1,41 @@
-# write your code here
+import pytest
+from unittest.mock import patch
+from app.main import can_access_google_page
+
+
+@pytest.mark.parametrize(
+    "url_status,internet_status,expected_result",
+    [
+        pytest.param(
+            True,
+            True,
+            "Accessible",
+            id="two parameters are correct"
+        ),
+        pytest.param(
+            False,
+            True,
+            "Not accessible",
+            id="url status is False parameters are correct"
+        ),
+        pytest.param(
+            True,
+            False,
+            "Not accessible",
+            id="internet status is False parameters are correct"
+        )
+    ]
+)
+@patch("app.main.valid_google_url")
+@patch("app.main.has_internet_connection")
+def test_can_access_google_page(
+    mock_internet_connection: bool,
+    mock_valid_url: bool,
+    internet_status: bool,
+    url_status: bool,
+    expected_result: str
```

Comment:
The order of the parameters `mock_internet_connection` and `mock_valid_url` in the function signature should match the order in the `@pytest.mark.parametrize` decorator. The correct order should be `url_status, internet_status, expected_result` to align with the parameterized test inputs.